adaptive_scheduler/monitoring/seeing.py

```diff
@@ -0,0 +1,68 @@
+import abc
+
+from adaptive_scheduler.monitoring.opensearch_seeing import get_seeing
+
+
+
+class SeeingMonitor(object):
```

Comment:
I'm not sure what version of Python we're on here, but this is a thing for later versions:

```python
from abc import ABC

class MyABC(ABC):
    pass
```

https://docs.python.org/3/library/abc.html